update example (#1)
KseniyaKuzmina/winforms-charts-create-a-pie-chart: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1189)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Chart for WinForms - Create a Pie Chart

The following example shows how to create a [Pie](https://docs.devexpress.com/WindowsForms/2978/controls-and-libraries/chart-control/series-views/2d-series-views/pie-and-donut-series-views/pie-chart) chart at runtime.

![Resulting chart](Images/resulting-chart.png)

> The Chart Control uses the [Simple Diagram](https://docs.devexpress.com/WindowsForms/5906/controls-and-libraries/chart-control/diagram/simple-diagram?p=netframework) to display pies. Cast the [ChartControl.Diagram](https://docs.devexpress.com/WindowsForms/DevExpress.XtraCharts.ChartControl.Diagram?p=netframework) property to the [SimpleDiagram](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraCharts.SimpleDiagram?p=netframework) type to access diagram settings. The Chart Control determines the diagram type based on the series that is added first. So you should refer to the diagram after at least one series is added to the chart. 

Reviewer: `So you should refer to the diagram` Why should you refer to the chart?

Author: Updated. Please review:
```suggestion
The Chart Control uses the [Simple Diagram](https://docs.devexpress.com/WindowsForms/5906/controls-and-libraries/chart-control/diagram/simple-diagram?p=netframework) to display pies. Cast the [ChartControl.Diagram](https://docs.devexpress.com/WindowsForms/DevExpress.XtraCharts.ChartControl.Diagram?p=netframework) property to the [SimpleDiagram](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraCharts.SimpleDiagram?p=netframework) type to access diagram settings. The Chart Control determines the diagram type based on the series that is added first. So we recommend that you access the diagram to configure its settings after at least one series is added to the chart.
```

Reviewer: ```suggestion
The Chart Control uses the [Simple Diagram](https://docs.devexpress.com/WindowsForms/5906/controls-and-libraries/chart-control/diagram/simple-diagram?p=netframework) to display pies. Cast the [ChartControl.Diagram](https://docs.devexpress.com/WindowsForms/DevExpress.XtraCharts.ChartControl.Diagram?p=netframework) property to the [SimpleDiagram](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraCharts.SimpleDiagram?p=netframework) type to access diagram settings. The Chart Control determines the diagram type based on the series that is added first. We recommend that you access the diagram to configure its settings after at least one series is added to the chart. 
```
